2-mandatory/3-stocks.js

```diff
@@ -48,9 +70,15 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+    return closingPricesForAllStocks.map(prices => { // we use the map method to intenerate each element of closingPricesForAllStock
```

Comment:
Very nice and short implementation 😄 
Again, you might need the `parseFloat` on lines 74 and 75.